### PR TITLE
Shifted to AppCompat, Removed ABS, Renamed privly applications as per new decisions

### PR DESCRIPTION
**Pull request by gitanshu**

### CODE REVIEW

**Comment by vshivam** [comment]
Can you upgrade the ActionBar to the Toolbar. Since, we're doing it let's use the latest. 

I already have an implementation here : https://github.com/vshivam/privly_v2/blob/master/app/src/main/res/layout/activity_home.xml#L13

**Comment by gitanshu** [comment]
Actually I am using a toolbar, the implementation style is a little different though.
https://github.com/gitanshu/privly-android/blob/4ebe9a4c1b8adf5bcd8cc8e2165f8a8cc3663f20/privlyandroid/src/main/res/layout/activity_main.xml

I'll look into the difference and change accordingly.

**Comment by vshivam** [comment]
Yep, I think this is how we should be implementing it : http://developer.android.com/training/implementing-navigation/nav-drawer.html#DrawerLayout

File: privlyandroid/src/main/res/values/themes.xml [hunk 1]
```diff
@@ -1,8 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
-    <style name="customPrivlyTheme" parent="Theme.Sherlock.Light">
+    <style name="customPrivlyTheme" parent="Theme.AppCompat.Light">
         <item name="android:windowBackground">@color/privlyBackgroundColor</item>
     </style>
 
+    <style name="DrawerArrowStyle" parent="Widget.AppCompat.DrawerArrowToggle">
```
Comment (vshivam) [inline privlyandroid/src/main/res/values/themes.xml]: Do we need this explicitly ? All of this works by default for me.

**Comment by vshivam** [comment]
Can you also update your .gitignore file to not include idea related files ?
This looks like a good one : https://gist.github.com/iainconnor/8605514 

Correct me if there's something in this that needs to be there. I do not have a lot of experience with android studio.

File: privlyandroid/src/main/res/values/themes.xml (hunk 1 above)
Comment (gitanshu) [inline privlyandroid/src/main/res/values/themes.xml]: I guess https://github.com/vshivam/privly_v2/blob/master/app/src/main/res/layout/activity_home.xml#L19 must be taking care of it. I have updated the code accordingly.

**Pull request merged**